src/isomorphic/classic/element/__tests__/ReactElementValidator-test.js

```diff
@@ -283,7 +286,7 @@ describe('ReactElementValidator', function() {
   });
 
   it('gives a helpful error when passing null, undefined, boolean, or number',
-      () => {
```

Comment:
This is a workaround for eslint/eslint#2239. It had @zpao and I scratching our heads for a bit.